ingest-app/src/cmr/ingest/data/bulk_update.clj

```diff
@@ -136,9 +136,7 @@
                  (concat (map #(vector task-id % "PENDING") concept-ids) [:transaction? false]))
           task-id))
       (catch Exception e
```

Comment:
We don't have to catch and throw the exception if we are not going to do anything with it.